src/emulator/gui/src/game_selector.cpp

```diff
@@ -179,22 +181,98 @@ void draw_game_selector(GuiState &gui, HostState &host) {
+                    if (ImGui::BeginPopupContextItem("#game_context_menu")) {
+                        if (ImGui::MenuItem("Boot", game.title.c_str()))
+                            selected[0] = true;
+                        if (ImGui::BeginMenu("Copy Game Info")) {
+                            if (ImGui::MenuItem("ID and Name")) {
+                                ImGui::LogToClipboard();
+                                ImGui::LogText("%s [%s]", game.title_id.c_str(), game.title.c_str());
+                                ImGui::LogFinish();
+                            }
+                            if (ImGui::MenuItem("ID")) {
+                                ImGui::LogToClipboard();
+                                ImGui::LogText("%s", game.title_id.c_str());
+                                ImGui::LogFinish();
+                            }
+                            if (ImGui::MenuItem("Name")) {
+                                ImGui::LogToClipboard();
+                                ImGui::LogText("%s", game.title.c_str());
+                                ImGui::LogFinish();
+                            }
+                            ImGui::EndMenu();
+                        }
+                        if (ImGui::BeginMenu("Remove")) {
+                            if (ImGui::MenuItem("Game"))
+                                gui.delete_game_popup = true;
+                            if (ImGui::MenuItem("Shader Log")) {
+                                if (fs::exists(shaderlog_path / game.title_id))
+                                    fs::remove_all(shaderlog_path / game.title_id);
+                            }
+                            ImGui::EndMenu();
+                        }
+                        ImGui::EndPopup();
+                    }
+
+                    auto games_index = std::find_if(gui.game_selector.games.begin(), gui.game_selector.games.end(), [game](Game g) {
+                        return g.app_ver == game.app_ver && g.title == game.title && g.title_id == game.title_id;
+                    });
+
+                    if (!fs::exists(game_path / gui.title_id)) {
+                        gui.game_selector.games.erase(games_index);
+                        if (gui.delete_game_popup)
+                            gui.delete_game_popup = false;
+                    }
+
+                    if (gui.delete_game_popup && fs::exists(game_path / gui.title_id))
+                        ImGui::OpenPopup("Delete Game");
+                    ImGui::PushStyleColor(ImGuiCol_Text, GUI_COLOR_TEXT_MENUBAR_OPTIONS);
+                    if (ImGui::BeginPopupModal("Delete Game", NULL, ImGuiWindowFlags_AlwaysAutoResize)) {
+                        ImGui::PopStyleColor();
+                        ImGui::Text("Do you really want delete this game ?\n%s [%s]", game.title_id.c_str(), game.title.c_str());
+                        ImGui::Separator();
+                        if (ImGui::Button("Yes", ImVec2(124, 0))) {
```

Comment:
No hardcode vec2. Remove or make it const

Reply:
make it constant ? how ? i have just set this like explain on imgui demo, for no have very small button ^^